src/herder/TxQueueLimiter.cpp

```diff
@@ -103,11 +103,14 @@ TxQueueLimiter::canAddTx(TransactionFrameBasePtr& newTx,
     auto txBaseFee = newTx->getBaseFeeBid();
     if (txBaseFee <= getMinBaseFee())
     {
-        return false;
+        auto minFee = newTx->getFee(Config::CURRENT_LEDGER_PROTOCOL_VERSION,
```

Comment:
Nitpick: Calling `getFee` is a little overkill here--I had to go check why we were using that function specifically. I would just do the multiplication explicitly here (after all, that's what happening inside of `getFee`) but it will avoid passing the version number, which is ignored anyway because you are passing `false` as the last argument.

Reply:
got rid of that